src/controllers/posts.ts

```diff
@@ -0,0 +1,87 @@
+import { Request, Response } from 'express';
+import axios from 'axios';
+import { ServerPostData } from '../interfaces/serverPostData';
+import { ServerPostArray } from '../interfaces/serverPostArray';
+import { ServerUserArray } from '../interfaces/serverUserArray';
+import { ServerUserElement } from '../interfaces/serverUserElement';
+import { User } from '../models/users';
+import { Posts } from '../models/posts';
+
+
+export class PostController {
+    domain:string = "https://jsonplaceholder.typicode.com/";
+    res:Response;
+    req:Request;
+
+    constructor (res:Response, req: Request) {
+        this.res = res;
+        this.req = req;
+    }
+
+convertPosts(paramUser: ServerUserElement,paramPostArray: ServerPostArray):Array<any> {
+    //console.log(paramUserArray.address)
+    let responseArray = paramPostArray.data.map((element)=>{
```

Comment:
User `lint` to find out what `let`s need to be turned into `const`.

Reply:
done